app/subscriber/src/features/my-reports/styled/MyReports.tsx

```diff
@@ -28,6 +28,14 @@ export const MyReports = styled.div`
     }
   }
 
+  .page-section {
+    margin-top: 0;
+  }
+
+  .my-reports-content {
+    height: calc(100dvh - 260px);
```

Comment:
`dvh` is handy for mobile, as it accounts for potential UI from things like the Safari app showing the URL bar at the top of the screen